README.md

```diff
@@ -27,6 +27,36 @@ Installers
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+#### Debian Repository
+
+- Save the following as `/etc/apt/sources.list.d/etcher.list`:
+
+```
+deb https://dl.bintray.com/resin-io/debian stable etcher
+```
+
+- Trust Bintray.com's GPG key:
```

Comment:
Indent the code blocks by four spaces to make them part of each `li`. Right now you have three separate list items with code between them.